From a8ea09dd041f7dedba1cc744f7639b66284a1703 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Wed, 11 Apr 2012 14:58:05 +0000
Subject: [PATCH 0187/1000] Mantis 0022440: Reduce the number of compiler
 warnings #pragma warning(disable:4290) // A function was declared using
 exception specification.

git-svn-id: svn://localhost/SelfServiceCommon/trunk@317 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Interfaces/generic/fsm/mStateMachine.hpp | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/SelfServiceCommon/Interfaces/generic/fsm/mStateMachine.hpp b/SelfServiceCommon/Interfaces/generic/fsm/mStateMachine.hpp
index 6ecea5e0..80a166a9 100644
--- a/SelfServiceCommon/Interfaces/generic/fsm/mStateMachine.hpp
+++ b/SelfServiceCommon/Interfaces/generic/fsm/mStateMachine.hpp
@@ -43,6 +43,11 @@
  *
  **************************************************************************
 **/
+
+#pragma warning(disable:4290) // A function was declared using exception specification.
+
+
+
 #ifndef MSTATEMACHINE_HPP
 #define MSTATEMACHINE_HPP
 #ifdef SHOW_INCLUDE_FILES
-- 
2.41.0.windows.1

